packages/http-client-java/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@typespec/http-client-java",
+  "version": "0.1.0",
+  "description": "TypeSpec library for emitting Java client from the TypeSpec REST protocol binding",
+  "keywords": [
+    "TypeSpec"
+  ],
+  "author": "Microsoft Corporation",
+  "homepage": "https://github.com/Microsoft/typespec",
+  "readme": "https://github.com/Microsoft/typespec/blob/main/packages/http-client-java/README.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/typespec.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/typespec/issues"
+  },
+  "license": "MIT",
+  "type": "module",
+  "main": "emitter/dist/src/emitter.js",
+  "exports": {
+    ".": "./emitter/dist/src/emitter.js",
+    "./testing": "./dist/src/testing/index.js"
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "clean": "rimraf .emitter/dist ./temp",
+    "build": "npm run build:emitter && npm run build:generator && npm run extract-api",
+    "build:emitter": "tsc -p ./emitter/tsconfig.build.json",
+    "build:generator": "mvn clean install -f ./generator/pom.xml",
+    "format": "pnpm -w format:dir packages/http-client-java",
+    "lint": "eslint . --max-warnings=0",
+    "lint:fix": "eslint . --fix",
+    "watch": "tsc -p ./emitter/tsconfig.build.json --watch",
+    "extract-api": "npx api-extractor run --local --verbose"
+  },
+  "files": [
+    "lib/*.tsp",
+    "emitter/dist/**",
+    "!emitter/dist/test/**",
+    "generator/http-client-generator/target/emitter.jar"
+  ],
+  "peerDependencies": {
+    "@azure-tools/typespec-autorest": ">=0.43.0 <1.0.0",
+    "@azure-tools/typespec-azure-core": ">=0.43.0 <1.0.0",
+    "@azure-tools/typespec-client-generator-core": ">=0.43.2 <1.0.0",
+    "@microsoft/api-extractor": "^7.40.3",
+    "@typespec/compiler": ">=0.57.0 <1.0.0",
+    "@typespec/http": ">=0.57.0 <1.0.0",
+    "@typespec/openapi": ">=0.57.0 <1.0.0",
```

Comment:
Same with openapi

Reply:
Will be investigated as part of this issue - https://github.com/microsoft/typespec/issues/4117